_inc/lib/debugger/class-jetpack-cxn-tests.php

```diff
@@ -746,7 +746,7 @@ protected function test__sync_health() {
 					esc_html__( 'Error', 'jetpack' )
 				);
 				$description .= wp_kses(
-					__( 'Jetpack has detected an error while syncing your site. <strong>We recommend <a id="full_sync_request_link" href="#">a full sync</a> to align Jetpack with your site data.</strong>', 'jetpack' ),
+					__( 'Jetpack has detected that data is not properly in sync which may be impacting some of your site\'s functionality. <strong>We recommend <a id="full_sync_request_link" href="#">a full sync</a> to align Jetpack with your site data.</strong>', 'jetpack' ),
```

Comment:
How about using ’ instead of the escaped single quote?

```suggestion
					__( 'Jetpack has detected that data is not properly in sync which may be impacting some of your site’s functionality. <strong>We recommend <a id="full_sync_request_link" href="#">a full sync</a> to align Jetpack with your site data.</strong>', 'jetpack' ),
```